pages/_app.page.js

```diff
@@ -1,4 +1,5 @@
 import "../styles/globals.css";
+import style from "../styles/contactus.css";
```

Comment:
Import it in contact.page.js

Reply:
> A couple of changes and looks good to merge. But we have to work on UI. It is not looking good

Okie, definitely I will change UI as per your requirements.

Reply:
It was giving me an error, that why I wrote it on _app.js .

Reply:
![gre](https://github.com/anmode/grabtern-frontend/assets/132778037/6924c1d6-b2a7-4013-87c7-04426075abc2)

Reply:
oh ok! np. I'll see but see for UI. It is not looking good to me :(

Reply:
> oh ok! np. I'll see but see for UI. It is not looking good to me :(

Okie, I'll change it. But can you tell me what kind of changes you want, so accordingly I will change it.